### PR TITLE
[sc-7575] Block all features on contributor app if no NUA key is provided

### PR DESCRIPTION
**Pull request by operramon**

### CODE REVIEW

File: libs/common/src/lib/navbar/navbar.component.html [hunk 1]
```diff
@@ -23,7 +23,8 @@
             [routerLink]="kbUrl + '/upload'"
             class="nav-link"
             routerLinkActive="active"
-            [routerLinkActiveOptions]="{ exact: true }">
+            [routerLinkActiveOptions]="{ exact: true }"
+            [class.disabled]="!(isAdminOrContrib | async) || (invalidKey | async)">
```
Comment (mpellerin42) [inline libs/common/src/lib/navbar/navbar.component.html:27]: I don't see how a simple css class can prevent user to click on the link and navigate

Comment (operramon) [reply libs/common/src/lib/navbar/navbar.component.html:27]: mm.. It prevents clicking it with `pointer-events: none`, but I see it doesn't disable keyword interaction

Comment (mpellerin42) [reply libs/common/src/lib/navbar/navbar.component.html:27]: ok, it's a bit corner case then, but you can make it fully disabled by adding a condition in `routerLink` as well, like 

```ts
[routerLink]="(isAdminOrContrib | async) === true && (invalidKey | async) === false ?  kbUrl + '/upload' : null"
```

Comment (operramon) [reply libs/common/src/lib/navbar/navbar.component.html:27]: Fixed

**Review by ebrehault — APPROVED** [review]

**Pull request merged**